### PR TITLE
make download button text dynamic based on purchase

### PR DESCRIPTION
**Pull request by btkostner**
It will now say 'Download elementary OS' or 'Purchase elementary OS' based on what you select

### CODE REVIEW

File: scripts/homepage.js [hunk 1]
```diff
@@ -224,5 +224,21 @@ $(function () {
         }, 3000);
     });
 
+    // Change Button text on payment click
+    $(function() {
+      function updateDownloadButton () {
+        var payment_amount = $('#amount-custom').val() * 100;
+        if ($('input#amount-custom').hasClass('checked') && payment_amount <= payment_minimum) {
```
Comment (lewisgoddard) [inline scripts/homepage.js]: It's `payment_amount < payment_minimum`, not `<=`

File: scripts/homepage.js [hunk 2]
```diff
@@ -224,5 +224,21 @@ $(function () {
...
+          $('#download').html('Download elementary OS');
```
Comment (lewisgoddard) [inline scripts/homepage.js]: Should probably use `.text()` instead of `.html()` seeing as it's just text.

File: scripts/homepage.js [hunk 3]
```diff
@@ -224,5 +224,21 @@ $(function () {
...
+        } else {
+          $('#download').html('Purchase elementary OS');
+        }
+      }
+
+      $('#amounts').on('click', updateDownloadButton);
+      $('#amounts input').on('change', updateDownloadButton);
+      $('button#download').on('focus', updateDownloadButton);
```
Comment (lewisgoddard) [inline scripts/homepage.js]: I'm not sure why we need this last event listener.

File: scripts/homepage.js [hunk 4]
```diff
@@ -224,5 +224,21 @@ $(function () {
         }, 3000);
     });
 
+    // Change Button text on payment click
+    $(function() {
```
Comment (lewisgoddard) [inline scripts/homepage.js]: Does this need to be within _another_ anonymous function?

File: scripts/homepage.js [hunk 5]
```diff
@@ -224,5 +224,21 @@ $(function () {
...
+      function updateDownloadButton () {
```
Comment (lewisgoddard) [inline scripts/homepage.js]: Indentation should be 4 spaces, not 2.

File: scripts/homepage.js (hunk 4 above)
Comment (btkostner) [inline scripts/homepage.js]: It doesn't but it seems to be all the rage in the file

**Pull request merged**